src/common/time/src/timestamp.rs

```diff
@@ -430,9 +430,20 @@ impl FromStr for Timestamp {
 fn naive_datetime_to_timestamp(
     s: &str,
     datetime: NaiveDateTime,
+    timezone: Option<&Timezone>,
 ) -> crate::error::Result<Timestamp> {
-    Timestamp::from_chrono_datetime(Utc.from_utc_datetime(&datetime).naive_utc())
-        .context(ParseTimestampSnafu { raw: s })
+    if timezone.is_none() {
```

Comment:
ditto

Reply:
Addressed in https://github.com/GreptimeTeam/greptimedb/pull/3175/commits/7c3bd196d83f285370940e0cce125e5a3bf533a5